Update README to link to docs (#5)
talkjs/talkjs-react: README.md

# React components for TalkJS

> This library makes it easy to use [TalkJS](https://talkjs.com) pre-built chat UIs inside a React web application.

Reviewer: Two small suggestions:
- Maybe say "The `@talkjs/react` library" here, to make it super explicit in the opening sentence?
- Maybe could remove "pre-built chat UIs", so that it becomes just "...makes it easy to use [TalkJS](https://talkjs.com) inside..."? As it makes it easy to use the entire TalkJS offering, not only the pre-built chat UI bit?

Author: I've gone with https://github.com/talkjs/talkjs-react/pull/5/commits/072c5e919e49abde24ca6bb1ac68b3f306403479, wanted to still emphasise that the library is mainly for the chat UI components. Hopefully clearer now!

> `@talkjs/react` encapsulates `talkjs`, the framework-independent [TalkJS JavaScript SDK](https://talkjs.com/docs/Reference/JavaScript_Chat_SDK). It provides React components for UI-related matters. For anything related to data manipulation, such as synchronizing user data, or creating and joining conversations, use the regular SDK.

Reviewer: Perhaps here at the end "... use the regular JavaScript SDK", again just to have it super explicit?

Author: Good idea, actually switched 'regular' -> 'JavaScript' as I think that's even better https://github.com/talkjs/talkjs-react/pull/5/commits/4d8aa57813ea6fde444d39e2606fc3fef3f9e9c1

TypeScript bindings are included.

> If you encounter any problems with `@talkjs/react`, please open an issue. If you have a problem with TalkJS itself, or if you're not sure where the problem lies, it's better to open a chat with our support on https://talkjs.com/ (just open the chat widget). TalkJS support is staffed by engineers.

Reviewer: ```suggestion
If you encounter any problems with `@talkjs/react`, please open an issue. If you have a problem with TalkJS itself, or if you're not sure where the problem lies, it's better to open a [chat for support](https://talkjs.com/?chat). TalkJS support is staffed by engineers.
```
I believe if you use this link then it automatically opens the chat window :)

Author: Nice, TIL!

## Prerequisites

- A [TalkJS account](https://talkjs.com/dashboard/login). TalkJS provides a ready-to-use chat client for your application. Your account gives you access to TalkJS's free development environment.
- A [React app](https://react.dev/learn/start-a-new-react-project) that you will add TalkJS to

## Documentation

- [Getting started guide](/Getting_Started/Frameworks/React/)
- [React SDK reference docs](/Reference/React_SDK/Installation/)
- [Team chat tutorial](https://talkjs.com/resources/how-to-use-talkjs-to-create-a-team-chat-with-channels/) and [example code](https://github.com/talkjs/talkjs-examples/tree/master/react/remote-work-demo)

## Examples

> The following examples use the [`Session`](/Reference/React_SDK/Components/Session/) and [`Chatbox`](/Reference/React_SDK/Components/Chatbox/) components from the React SDK to create a chatbox UI.

Reviewer 2: I believe we need to change the urls into absolute urls since welink from github towards our docs

Author: Thanks, done now
